From 318c559411930f734f67d60165f271e7aede9b0c Mon Sep 17 00:00:00 2001
From: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Date: Tue, 7 May 2019 19:00:24 +0300
Subject: [PATCH 06/22] configs: introduce xt_defconfig

This is default configuration file for xen-troops Renesas boards

Signed-off-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 xen/arch/arm/configs/xt_defconfig | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 xen/arch/arm/configs/xt_defconfig

diff --git a/xen/arch/arm/configs/xt_defconfig b/xen/arch/arm/configs/xt_defconfig
new file mode 100644
index 0000000000..379bd5b324
--- /dev/null
+++ b/xen/arch/arm/configs/xt_defconfig
@@ -0,0 +1,6 @@
+CONFIG_XSM=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+CONFIG_IPMMU_VMSA=y
+CONFIG_IOREQ_SERVER=y
+CONFIG_EXPERT=y
-- 
2.34.1

